stle.css

```diff
@@ -0,0 +1,3 @@
+header {
+  font-size: 12px;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/59416136/153187692-dfead613-3c54-4182-b5e5-ba4a347f78fd.png)

- [ ] Great job on implementing other CSS properties i.e `font-size` however, it is one of the requirements to use your favourite colour.
- [ ] Please use proper tag name when you're styling, kindly make sure the tag you're styling exists in the HTML file.